wasm/lib/src/wasm_builtins.rs

```diff
@@ -31,15 +32,52 @@ pub fn print_to_html(text: &str, selector: &str) -> Result<(), JsValue> {
 }
 
 pub fn format_print_args(vm: &mut VirtualMachine, args: PyFuncArgs) -> Result<String, PyObjectRef> {
+    // Handle 'sep' kwarg:
```

Comment:
This is unrelated to this pull request, but I feel we are doing something wrong by having the print function twice in our codebase. Should we use `sys.stdout` to print to? Should we use file handles somehow?

Reply:
As I hinted in another comment – I think I have an idea how to sort printing in a generic way out, I’ll probably try to do it in another PR which I think I’ll open later this week. :)

Reply:
Cool! This would be some good improvements!